lint/linter.py

```diff
@@ -1118,7 +1117,7 @@ def clear_view(view):
         if not view:
             return
 
-        view.erase_status(STATUS_KEY)
+        status_bar_view.clear(view)
```

Comment:
We resolve this as we do #899 No change needed here in the mean time

Reply:
I don’t even think cleaning the status bar here is necessary actually, it worked just as well when I removed this line. Let’s do that.